my_project/tab_sun/app_sun.py

```diff
@@ -109,10 +110,11 @@ def explore_daily_heatmap():
         className="container-col full-width",
         children=[
             html.Div(
-                children=title_with_tooltip(
+                children=title_with_link(
                     text="Daily charts",
-                    tooltip_text=None,
+                    tooltip_text="Click to learn more about the Daily charts",
```

Comment:
actually we should be more consistent sometimes we say:
* to learn more
* to visualize

I think we should be consistent and say. "Click to access the official documentation" No need to repeat the name of the chart since it is already showing on the page.